src/index.html

```diff
@@ -11,7 +11,176 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form class="form" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label for="surname" class="label">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            class="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label for="name" class="label">
+          Name:
+          <input
+            type="text"
+            name="name"
+            id="name"
+            class="name"
+            autocomplete="off"
+          >
+        </label>
+        <label for="age" class="label">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            id="age"
+            class="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label for="dob" class="label">
+          Full date of birth:
+          <input
+            type="date"
+            name="dob"
+            class="dob"
+            id="dob"
+          >
+        </label>
+        <label for="terms" class="label">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+            class="terms"
+            id="terms"
+            value="agree"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label for="email" class="label">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            id="email"
+            class="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label for="password" class="label">
+          Password:
+          <input
+            type="password"
+            name="password"
+            id="password"
+            class="password"
+            minlength="6"
+            maxlength="18"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label for="radio" class="label">
+          Do you love cats?
+          <input
+            type="radio"
+            name="loveCats"
+            id="loveCatsYes"
+            class="radio"
+            value="yes"
+          >Yes
```

Comment:
better to move text to a new line:
```suggestion
          >
          Yes
```